### ISSUE TITLE
Tanaris Field Sampling never fails

### ISSUE THREAD

**Issue by Simp-N** [TeamEverlook/Everlook-BugTracker#374]
tl;dr Tanaris Field Sampling is bugged and never fails. I think the success rate should be at somewhere around 1/3 or 2/5.

For the quest [Tanaris Field Sampling](https://classicdb.ch/?quest=654) you have to gather 8x3 accepted samples from beasts in Tanaris. These accepted samples are created by using untested samples that you get from killing [scorpids](https://classicdb.ch/?item=9442), [hyenas](https://classicdb.ch/?item=9439), and [basilisks](https://classicdb.ch/?item=9437). When you use these samples, there's supposed to be a chance to get an accepted one if its successful, or get nothing if it fails. Currently on Everlook the success rate of these items are 100%, as in you never fail the test, making what's supposed to be a quite grindy quest very short and easy (took me ten minutes to complete). 

I'm not 100% sure what the success rate should actually be, but I tried going through comments on Wowhead, as well as counting the number of samples needed by Joana in his vanilla speedrun to see about how much it should be. On Wowhead you'll see quite a few people mention the horrible drop rate of the accepted samples. Most of them dont mention any hard numbers, and some just mention rough estimates, such as 1/5. A few people also mention that the success rate might be related to the level of the mob that dropped the sample, but I doubt that's true simply because they all drop the same item and I doubt the item itself can "remember" the exact mob that dropped it. People also mention how long it took to complete the quest and it varies a bit. Below I'll post some comments from [Wowhead's page of the quest](https://www.wowhead.com/classic/quest=654/tanaris-field-sampling#comments):
> This is what it took for me to finish it
>
>Starving Blisterpaw : 27
>Glasshide Basilisk : 14
>Scorpid Hunter : 19
>Scorpid Tail Lasher : 2
>
>Took almost the full 2 hours as a level 41 shadow priest. Got 4050xp from the gadetzan turn in.

By this, we get:
8/27 = 0,30
8/14 = 0,57
8/21 = 0,38
or 24/62 = **0,39**
>From my experience, you'll probably need around 25-30 untested samples to get the 8 you need.  

That's a little less than 1/3.

>The conversion rate from untested to acceptable is less than **10%**. Which, if it was exactly 10%, would be 240 monsters.

>Went for the lvl 47+ mobs and found that the conversion of untested to acceptable findings were about **1:5**.

>The rate in which the untested samples turned to accepted samples was frustrating, maybe **1 out of 6** was accepted.

Like I said, I also watched through [Joana](https://youtu.be/KO1iKRoKDyI&t=4660) completing this quest. It took him about 45 minutes to complete, but then he had samples to spare, and the number of samples he used was
20 scorpid (8/20 = 0,4)
28 hyena (8/28 = 0,28)
23 basilisk (8/23 = 0,35)
or 24/71 = **0,34**

I would guess that some of the people that just mention drop rates in 1:5, for example, are putting it a bit worse than it actually is. Looking at the numbers I could find it appears to be slightly better than 1/3. Perhaps more like 2/5, or 40%. But its difficult since I dont have a too great sample size. At any rate, 100% success rate is way too high, bugged simply, and trivializes the quest. And even if we can't get the perfect, correct, vanilla-like success rate, I think we can at least make it better than what it currently is.
